woocommerce/woocommerce-android
# Issue/2629 edit product sku

Fixes #2629 by adding logic to edit the SKU field for external and grouped products.

#### Changes
- Added logic to display only the SKU field in the product inventory screen, if the product type is external and grouped.
- Also modified the order of the product fields displayed in the product detail screen, based on the designs.
  - SIMPLE products: Price, product type, inventory, categories, tags, short description.
  - GROUPED products: Product type, SKU, categories, tags, short description.
  - EXTERNAL products: Price, Product type, external link, SKU, categories, tags, short description.

#### Screenshots
<img src="https://user-images.githubusercontent.com/22608780/88621135-5152d200-d0bd-11ea-839e-421b50f66753.png" width="300"/>. <img src="https://user-images.githubusercontent.com/22608780/88621139-531c9580-d0bd-11ea-9590-7db7e4747e95.png" width="300"/>. <img src="https://user-images.githubusercontent.com/22608780/88621148-557eef80-d0bd-11ea-9136-e91a79a197cc.png" width="300"/>

#### Testing
- Click on a SIMPLE product from the Products tab and verify that the order of the product section is displayed correctly.
- Click on a EXTERNAL product from the Products tab and verify that the order of the product section is displayed correctly.
  - Click on the Inventory section and try to update the SKU field and verify that the field is updated successfully.
- Click on a GROUPED product from the Products tab and verify that the order of the product section is displayed correctly.
  - Click on the Inventory section and try to update the SKU field and verify that the field is updated successfully.

Update release notes:
